.github/workflows/ci.yml

```diff
@@ -72,10 +72,26 @@ jobs:
           toolchain: ${{ matrix.toolchain }}
           target: thumbv7em-none-eabihf
           override: true
+          components: llvm-tools-preview
+      - name: Install binary utils
+        uses: actions-rs/cargo@v1
+        with:
+          command: install
+          args: |
+            cargo-binutils
       - uses: actions-rs/cargo@v1
         with:
           command: build
           args: --release --features "${{ matrix.features }}"
+      - name: Objcopy release binary
+        uses: actions-rs/cargo@v1
+        with:
+          command: objcopy
+          args: --release --bin thermostat-eem -- -O binary ${{ matrix.toolchain }}-thermostat-eem.bin
```

Comment:
Why the toolchain in the name?
You'd only want the upload for a single toolchain.
But instead potentially and likely for multiple features.

Reply:
Well, I didn't know which binary is regarded to be official. For now I left only stable binary and moved cargo binutils under condition, so it would consume a bit less time quote.